src/main/java/org/traccar/protocol/Gl200TextProtocolDecoder.java

```diff
@@ -148,6 +188,7 @@ private Position decodeAck(Channel channel, SocketAddress remoteAddress, String[
             .expression("(?:[01])?,").optional() // pin15 mode
             .number("(d+)?,")                    // adc1
             .number("(d+)?,").optional()         // adc2
+            .number("(d+)?,").optional()         // adc3
```

Comment:
I'm pretty sure this would break some use cases. Do we have tests for it?

Reply:
Yes, the last one.

Reply:
I think you misunderstood what I said. I'm asking for test cases for old stuff to make sure it doesn't break any case where adc3 is missing.

Reply:
for that I was relying on the existing tests.

Reply:
We need to make sure it's covered.